packages/fx-core/src/plugins/resource/bot/v2/plugin.ts

```diff
@@ -161,48 +138,21 @@ export class TeamsBotV2Impl {
     return ok(Void);
   }
 
-  private getTemplates(ctx: Context, inputs: Inputs): CodeTemplateInfo[] {
-    const actRoles = this.resolveActRoles(ctx, inputs);
-    const triggers = this.resolveTriggers(inputs);
-    const hostType = this.resolveHostType(ctx);
-    const lang = this.resolveProgrammingLanguage(ctx);
-
-    const scenarios = this.resolveScenarios(actRoles, triggers, hostType);
-
-    return scenarios.map((scenario) => {
-      return {
-        group: TemplateProjectsConstants.GROUP_NAME_BOT,
-        language: lang,
-        scenario: scenario,
-        variables: {},
-      };
-    });
-  }
-
   private getBicepConfigs(ctx: Context, inputs: Inputs): BicepConfigs {
-    const lang = this.resolveProgrammingLanguage(ctx);
-
     const bicepConfigs: BicepConfigs = [];
-
-    if (lang === "js" || lang === "ts") {
-      bicepConfigs.push("node");
-    }
-    if (lang === "csharp") {
-      bicepConfigs.push("dotnet");
-    }
-
+    const lang = resolveProgrammingLanguage(ctx);
+    bicepConfigs.push(runtimeMap[lang]);
```

Comment:
Nit: Add error handling to improve robustness when `lang` is not found in runtimeMap.